package/MDAnalysis/analysis/base.py

```diff
@@ -281,26 +298,51 @@ def run(self, start=None, stop=None, step=None, verbose=None):
             stop frame of analysis
         step : int, optional
             number of frames to skip between each analysed frame
+        frames : array_like, optional
+            array of integers or booleans to slice trajectory
         verbose : bool, optional
             Turn on verbosity
+
+
+    Example
+    -------
```

Comment:
Should this be indented by two the right?

Reply:
You're right, I fixed it